WordPress/src/main/java/org/wordpress/android/ui/reader/SubfilterBottomSheetFragment.kt

```diff
@@ -89,4 +101,41 @@ class SubfilterBottomSheetFragment : BottomSheetDialogFragment() {
         super.onCancel(dialog)
         viewModel.setIsBottomSheetShowing(false)
     }
+
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    fun onEventMainThread(event: ReaderEvents.FollowedTagsChanged) {
```

Comment:
We might want to consider moving all these methods into a VM. Wdyt?

Reply:
yep, same refactor commit as above here feef4bceedb37674d9b5ef01706059c0837e6f9a